user-guide/modules/ROOT/nav.adoc

```diff
@@ -1,11 +1,19 @@
 * xref:intro.adoc[]
-* xref:getting-started.adoc[]
 * xref:whats-inside.adoc[]
-* xref:reporting-issues.adoc[]
-* xref:advanced-builds.adoc[]
-* xref:faq.adoc[]
-* xref:tutorials.adoc[]
-* xref:testing-debugging.adoc[]
+* xref:getting-started.adoc[]
+
+* Building 
+** xref:installing-select-libraries.adoc[]
```

Comment:
This content is already in "getting started". Both with package managers and our own scripts. The cons of another section could be that 1) it would cause the impression that installing select libraries is so difficult it needs another section, 2) someone could not get to this section or see that as a edge case that's not recommended, and 3) the getting started is no longer already answering the most common complaints about boost (being a monolith and not supporting cmake), which is one of the objectives there. This could be good if the content about it gets too large, but I don't think we are at this stage yet because the process is indeed quite simple.

Reply:
@alandefreitas Agreed, will remove this.